README.md

```diff
@@ -260,9 +260,11 @@ the specified context and less than the context for the next release.
 
 | jwst tag            | DMS build | SDP_VER  | CRDS_CONTEXT | Released   | Ops Install | Notes                                         |
 |---------------------|-----------|----------|--------------|------------|-------------|-----------------------------------------------|
-| 1.18.0rc1           | B11.3     | TBD      | 1350         | 2025-03-19 | TBD         | First release candidate for B11.3             |
-| 1.17.1              | B11.2     | TBD      | 1321         | 2025-01-02 | 2025-03-05  | Second release candidate for B11.2            |
-| 1.17.0              | B11.2     | TBD      | 1321         | 2024-12-20 | TBD         | First release candidate for B11.2             |
+| 1.18.0              | B11.3     | 2025.2.0 | 1364         | 2025-04-01 | 2025-05-20  | Final release for B11.3                       |
+| 1.18.0rc2           | B11.3     | 2025.2.0 | 1350         | 2025-04-01 | 2025-05-20  | Second release candidate for B11.3            |
```

Comment:
@tapastro did you say you want me to remove all traces of RC here?

Reply:
Yes, this is the public record I was referring to - I just don't think it's useful information to provide and clutters the table. You can leave them in, but maybe future releases will not list them. 🤷

Reply:
Gotcha. Removed!